README.md

```diff
@@ -306,3 +306,14 @@ make cluster-down
 # Run function test
 make functest
 ```
+
+In just the functest want to be deployed at pre-deployed external cluster
```

Comment:
How about this wording?
 "If you would like run e2e tests on an already deployed external cluster.."
or "In order to run e2e tests on external cluster.."